composer.json

```diff
@@ -10,13 +10,14 @@
     ],
     "prefer-stable": true,
     "require": {
-        "php": ">= 8.0.2",
+        "php": ">= 8.2",
         "algolia/algoliasearch-client-php": "^3.0",
-        "doctrine/event-manager": "^1.1",
+        "doctrine/annotations": "^2.0",
```

Comment:
I see that you moved a lot of items to PHP `Attributes` I think that this means that doctrine/annotations can be removed ([see comment on their README](https://github.com/doctrine/annotations)).

Reply:
Indeed, everything passes without it as well. Thanks for spotting!